### PR TITLE
Fixes #28636 - allow group_by on AR objects

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by tbrisker** [comment]
Won't permitting `:try` allow executing all methods as its argument, effectively bypassing safe mode?

**Comment by ares** [comment]
Worth testing for sure, I was thinking the internal implementation is using respond_to? and then calls the method, which should still work on Jail object. At the end, try may not be needed, so I can drop it from this PR also, when I get to computer.

**Comment by tbrisker** [comment]
Looks like it does. The template:
```erb
<%= @host.interfaces.try(:to_sql) %>
```
generates the output:
```sql
SELECT "nics".* FROM "nics" WHERE "nics"."host_id" = 50 ORDER BY "nics"."identifier" ASC
```
despite to_sql being disallowed in jail.

This is because `:try` actually executes `:public_send` if the the object responds to the method:

https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/object/try.rb#L15

**Comment by ares** [comment]
Great catch and thanks for the link, repushed without `:try`. Sadly the safe navigation operator does not seem to work in ERB. So we'll have many ifs :-)

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "url-loader": "^1.0.1",
     "webpack": "^3.4.1",
     "webpack-bundle-analyzer": ">=3.3.2",
-    "webpack-dev-server": "^2.5.1",
+    "webpack-dev-server": "2.10.1",
```
Comment (tbrisker) [inline package.json]: I don't think you meant to commit this change?

**Comment by tbrisker** [comment]
please rebase on develop, and update the commit message as well

**Comment by ares** [comment]
rebased and updated, should be ready

**Comment by tbrisker** [comment]
[test upgrade]

**Comment by tbrisker** [comment]
[test upgrade]

**Review by tbrisker — APPROVED** [review]
Thanks @ares !

**Pull request merged**